.github/ISSUE_TEMPLATE/question-issue.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Ask a question about or report an issue with GEOS-Chem
+---
+
+<!--- Before submitting an issue please see:
+
+- [Support guidelines](https://geos-chem.readthedocs.io/en/stable/reference/SUPPORT.html)
+- [User manuals](https://geos-chem.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/related-docs.html)
+- [Debugging GEOS-Chem and HEMCO errors](https://geos-chem.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/debug-guide.html)
+
+--->
+
+### Your name and institution
+
+We ask that users identify themselves and their institutions when submitting a new issue.
+
+### Description of your issue or question
+
+Please provide as much detail as possible and include the GEOS-Chem version, log files, etc.
```

Comment:
I think etc is too vague. What about just this? "Please provide as much detail as possible. Always include GEOS-Chem version number and any relevant files."

Reply:
Now done in commit 581e361a